src/api.v2/routes/kubernetes.js

```diff
@@ -0,0 +1,33 @@
+const AWSXRay = require('aws-xray-sdk');
+const k8s = require('@kubernetes/client-node');
+const getLogger = require('../../utils/getLogger');
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+
+const logger = getLogger();
+
+module.exports = {
+  'kubernetes#event': async (req, res, next) => {
+    logger.log('received kubernetes event');
+    try {
+      const {
+        reason, message, type, involvedObject: { name, namespace },
+      } = req.body;
+      logger.log(`[${reason}] received ${type} kubernetes event: ${message} ${name} in ${namespace}`);
+
+      // remove only pods in your namespace and due to backoff errors
+      if ((namespace.match('^pipeline-.*') || namespace.match('^worker-.*'))
```

Comment:
This string includes regex characters `^` and `.*`. Instead of a string, would it be clearer to change this string as a Regex? e.g. `/^pipeline-.*/`

Reply:
what's the difference? I think they do the same